doc/hardware/porting/board_porting.rst

```diff
@@ -34,39 +88,26 @@ You can visualize the hierarchy like this:
 Here are some examples. Notice how the SoC series and family levels are
 not always used.
 
-.. list-table::
-   :header-rows: 1
-
-   * - Board
-     - SoC
-     - SoC series
-     - SoC family
-     - CPU core
-     - Architecture
-   * - :ref:`nrf52dk/nrf52832 <nrf52dk_nrf52832>`
-     - nRF52832
-     - nRF52
-     - Nordic nRF5
-     - Arm Cortex-M4
-     - Arm
-   * - :ref:`frdm_k64f <frdm_k64f>`
-     - MK64F12
-     - Kinetis K6x
-     - NXP Kinetis
-     - Arm Cortex-M4
-     - Arm
-   * - :ref:`stm32h747i_disco <stm32h747i_disco_board>`
-     - STM32H747XI
-     - STM32H7
-     - STMicro STM32
-     - Arm Cortex-M7
-     - Arm
-   * - :ref:`rv32m1_vega_ri5cy <rv32m1_vega>`
-     - RV32M1
-     - (Not used)
-     - (Not used)
-     - RI5CY
-     - RISC-V
+.. table::
+
+   +--------------------------------------------+-----------------------+-------------+---------------+---------------+----------------+--------------+
+   | Board                                      | Identifier            | SoC         | SoC Series    | SoC family    | CPU core       | Architecture |
+   +============================================+=======================+=============+===============+===============+================+==============+
+   | :ref:`nrf52dk <nrf52dk_nrf52832>`          | /nrf52832             | nRF52832    | nRF52         | Nordic nRF5   | Arm Cortex-M4  | Arm          |
```

Comment:
family is actually nRF not nRF5

Reply:
actually I haven't changed that.
It's written as `Nordic nRF5` in the current docs:
https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#boards-socs-etc

but nRF is more correct, so updated.